sigar_windows.go

```diff
@@ -306,6 +312,45 @@ func GetProcName(pid int) (string, error) {
 
 }
 
+func GetProcCredName(pid int) (string, error) {
+	handle, err := syscall.OpenProcess(syscall.PROCESS_QUERY_INFORMATION, false, uint32(pid))
+
+	defer syscall.CloseHandle(handle)
+	
+	if err != nil {
+		return "", fmt.Errorf("OpenProcess fails with %v", err)
+	}
+	
+	var token syscall.Token
```

Comment:
x/sys/windows would definitely simplify the account lookup and it looks really bullet proof from an error perspective... perhaps we could just use it for GetProcGredName and defer the re-implementation of the other methods to a later date.